src/components/outline/Display.tsx

```diff
@@ -169,15 +178,20 @@ const Display: React.FC = () => {
           </Box>
         </Flex>
         <br></br>
-        <Box paddingLeft={"5%"} paddingRight={"5%"}>
-          <Text fontSize={32}>{title}</Text>
-          <br></br>
-          <Flex flexWrap="wrap" justifyContent="space-evenly">
-            {data?.hexathonUsers
-              .filter((hUser: any) => hUser.userId !== user?.uid)
-              .map((user: UserCardType) => <UserCard key={user.name} {...user} />)}
-          </Flex>
+        <Box display="flex" justifyContent="space-between" alignItems="center" borderWidth="1px" borderRadius="lg" borderColor="purple" width="200px" marginLeft={"auto"} marginRight={"auto"}>
+            <Button colorScheme='purple' variant={displayMode=="allUsers" ? "solid": "ghost"} width="100px" onClick={displayUsers}>
+                Individuals
+            </Button>
+            <Button colorScheme='purple' variant={displayMode=="allTeams" ? "solid": "ghost"} width="100px" onClick={displayTeams}>
+                Teams
+            </Button>
```

Comment:
for these button styles, could you make it look more like this? with extra padding and grey bg (just gave you access to figma)
![image](https://github.com/HackGT/match/assets/63804742/103eb7e3-5d71-44e2-9059-aecef45e698a)